packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswerContent/quanticGeneratedAnswerContent.js

```diff
@@ -0,0 +1,68 @@
+import MARKED_JS from '@salesforce/resourceUrl/marked';
+import {transformMarkdownToHtml} from 'c/quanticUtils';
+// @ts-ignore
+import {loadScript} from 'lightning/platformResourceLoader';
+import {LightningElement, api} from 'lwc';
+// @ts-ignore
+import generatedMarkdownContentTemplate from './templates/generatedMarkdownContent.html';
+// @ts-ignore
+import generatedTextContentTemplate from './templates/generatedTextContent.html';
+
+export default class QuanticGeneratedAnswerContent extends LightningElement {
+  @api isStreaming;
+  @api answerContentFormat = 'text/plain';
+  @api
+  get answer() {
+    return this._answer;
+  }
+  set answer(value) {
+    this._answer = value;
+    if (this.answerContentFormat === 'text/markdown') {
+      this.updateHtmlContent();
+    }
+  }
+
+  _ready = false;
+  _answer;
+
+  connectedCallback() {
+    if (this.answerContentFormat === 'text/markdown') {
+      loadScript(this, MARKED_JS + '/marked.min.js')
+        .then(() => {
+          this._ready = true;
+          this.updateHtmlContent();
+        })
+        .catch((error) => {
+          console.error('Error loading marked', error);
```

Comment:
```suggestion
          console.error('Error loading the Marked.JS library', error);
```

Reply:
If marked doesn't load we still display the answer?

Reply:
Should be handled now